.github/workflows/snyk_merge.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name: Snyk Clojure Merge
+
+on: push
+
+jobs:
+ snyk_clojure:
+   runs-on: ubuntu-latest
+   steps:
+    - name: checkout the current PR
```

Comment:
Should this say "branch head" since this is running on a push rather than on a PR?

Reply:
That's a screenshot for what I just added to puppetserver where I changed the name.